docs/developers.md

```diff
@@ -4,6 +4,32 @@
 
 See [interface.py], which defines the interface of a backend and may serve as a template for creating new backends.
 
+### Developing the sparqlwrapper
+
+Tripper comes with an inbuilt backend to the SPARQLWrapper. In order
+to test this properly a real triplestore is needed. This is not done in the
+automatic workflows on github. However, a lcal graphDB can be setup as described below and tested with test_sparqlwrapper_graphdb.py.
+
+To create the local instance of graphdb:
+```bash
+docker pull ontotext/graphdb:10.8.3 # latest tag 17.02.2025
+docker run -d -p 7200:7200 --name graphdb ontotext/graphdb:10.8.3
+```
+
+To create a repository in the graphdb isntance:
+```bash
+curl -X POST \
+     -H "Content-Type: application/rdf+xml" \
+     --data-binary @tests/backends/graphdb_config.xml \
+     http://localhost:7200/rest/repositories
```

Comment:
This doesn't work for me. Gives the following warnings:

    Warning: Couldn't read data from file "tests/backends/graphdb_config.xml", 
    Warning: this makes an empty POST.

Do you have a local non-committet file: `tests/backends/graphdb_config.xml`